R/stacking.R

```diff
@@ -0,0 +1,1626 @@
+#' @title Stack data set
+#' @description Stacks variables in a data set.
+#' @param input.data.set.name Name of data file to stack, either as a path to a
+#'   local file or file in the Displayr Cloud Drive
+#' @param stacked.data.set.name Name of the stacked data file to be saved in
+#'   the Displayr Cloud Drive (if run from Displayr) or saved locally.
+#' @param stack.with.common.labels \code{"Automatically"},
+#'   \code{"Using a set of variables to stack as reference"},
+#'   \code{"Using manually input common labels"},
+#'   \code{"Disabled"}.
```

Comment:
This parameter takes a string which comes directly from the GUI combobox corresponding to this parameter. So it has to be readable since users see this, and it helps us too.